### ISSUE TITLE
RangeError: Maximum call stack size exceeded. Issue with lambda toString method.

### ISSUE THREAD

**Issue by msvbg** [babel/babel#1179]
```
let f = x => f;
console.log(f(5));
```

The above code causes a `RangeError`, whereas the ES5 version

```
function f(x) { return f; }
console.log(f(5));
```

works fine. There seems to be an issue with infinite recursion in the lambda's `toString` method.

**Comment by alawatthe** [comment]
I think you encountered the bug #1164, which was fixed in #1170 and released yesterday in babel 5.0.9. So upgrading babel should fix it :-)

**Comment by msvbg** [comment]
Ah, thanks for the snappy response! That should do the trick.
